### ISSUE TITLE
cosine learning rate

### ISSUE THREAD

**Issue by tangbohu** [d-li14/mobilenetv2.pytorch#16]
Thanks for sharing your code! However, your implemenation of cosine learning rate seems not right.
In your implementation, the learing rate start with the initial learning rate (lr), but stop with 0.5*lr, instead of 0.
